①infoのナレッジインサイト〜採用情報までをcssでスペースを作りました。

NorikoAso opened the pull request:
②headerpictureのglass.svgとearth.svgの位置をcssで調整し、ハンバーガーメニューを作成しました。
③bread-listの文字をcssで画像の上に載せました。
④SNSbuttonをcssでスペースを作りました。
⑤NRImessageの部分でbrタグを使用し、見本と同じ状態に修正しました。

NorikoAso committed:
①infoのナレッジインサイト〜採用情報までをcssでスペースを作りました。
②headerpictureのglass.svgとearth.svgの位置をcssで調整し、ハンバーガーメニューを作成しました。
③bread-listの文字をcssで画像の上に載せました。
④SNSbuttonをcssでスペースを作りました。
⑤NRImessageの部分でbrタグを使用し、見本と同じ状態に修正しました。

pantomimer commented on index.html:
```diff
@@ -14,19 +14,19 @@
 <img src="img/nrilogo.png" alt="NRI" width=" 70" height="50"></div>
```
widthのあとの空白を取りましょう。

pantomimer commented on index.html:
```diff
@@ -37,36 +37,35 @@
      <!-- ここまで -->
      </div>
```
このdivはどこのdivでしょうか？

pantomimer reviewed (commented):
とりあえずHTMLのレビューです。
あと、&lt;h1&gt; から始まって&lt;h17&gt;まで存在するのですが、
そういうものではありません。
&lt;h1&gt;が大見出し、&lt;h2&gt;が中見出し、&lt;h3&gt;が小見出しぐらいまででとどめましょう。

普通は
&lt;h1&gt;大分類&lt;/h1&gt;
&lt;h2&gt;中分類&lt;/h2&gt;
&lt;h3&gt;トピック１&lt;/h3&gt;
&lt;h3&gt;トピック２&lt;/h3&gt;
&lt;h3&gt;トピック３&lt;/h3&gt;
&lt;h3&gt;トピック４&lt;/h3&gt;
のように使います。

一度
https://validator.w3.org/#validate_by_upload
を通してみてくださいね。

NorikoAso committed:
①widthのスペースを半角スペースに修正しました。
②h4-h9まで使用していたものをstrongタグに修正しました。
③TouTube→You Tubeに修正しました。
④h10-h17まで使用していたものをdivタグに変更しました。

NorikoAso commented:
①widthのスペースを半角スペースに修正しました。
②h4-h9まで使用していたものをstrongタグに修正しました。
③TouTube→You Tubeに修正しました。
④h10-h17まで使用していたものをdivタグに変更しました。
